### PR TITLE
adds new modalities

### LINKED ISSUE #218: add new modalities
We need to add:
- `HSFP`  - hyperspectral fiber photometry
- `FIP` - frame-injected independent-fiber photometry
- `MRI` - magnetic resonance imaging

### PR DESCRIPTION
**Pull request by dyf**
resolves #218 

also runs the linters

### CODE REVIEW

File: src/aind_data_schema/data_description.py [hunk 1]
```diff
@@ -44,31 +44,34 @@ class DataLevel(Enum):
...
+    OPHYS = "ophys"
 
 
 class Modality(Enum):
     """Data collection modality name"""
 
     ECEPHYS = "ecephys"
     EXASPIM = "ExASPIM"
-    SMARTSPIM = "SmartSPIM"
+    FIP = "FIP"
```
Comment (saskiad) [inline src/aind_data_schema/data_description.py:68]: Can you define was FIP is?

File: src/aind_data_schema/data_description.py [hunk 2]
```diff
@@ -44,31 +44,34 @@ class DataLevel(Enum):
...
+    FMOST = "fMOST"
+    HSFP = "HSFP"
```
Comment (saskiad) [inline src/aind_data_schema/data_description.py:70]: same here - it would be good to have some of these acronyms unpacked a bit more.

**Review by saskiad — CHANGES_REQUESTED** [review]
I want to clean up some of our acronyms when they aren't obvious. Can you define these modalities?

**Comment by dyf** [comment]
@saskiad can we add these now and handle adding long names in a separate PR.  This will impact our data entry form, I think, so will take some care to do properly.

**Review by saskiad — APPROVED** [review]

**Pull request merged**